modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-binding-workloads-that-are-not-compliant-with-PodSpec_{context}"]
+= Binding secondary workloads that are not compliant with PodSpec
+
+A typical scenario in service binding involves backing service, workload (Deployment), and {servicebinding-title}. Consider a scenario that involves a secondary workload, which can also be an application Operator, not compliant with PodSpec and is between the primary workload (Deployment) and {servicebinding-title}.
```

Comment:
```suggestion
A typical scenario in service binding involves configuring the backing service, the workload (Deployment), and {servicebinding-title}. Consider a scenario that involves a secondary workload (which can also be an application Operator) that is not compliant with PodSpec and is between the primary workload (Deployment) and {servicebinding-title}.
```
This is a little bit confusing to me as it's currently worded. I've made a few suggestions, but feel free to modify as needed if I've changed the meaning too much,

Reply:
Good catch! Thanks a lot for the suggestions.